frontend/components/MainPage/header/AgentButton/AgentNotRunningButton.tsx

```diff
@@ -192,7 +192,7 @@ const useServiceDeployment = () => {
           stakingProgramId: selectedStakingProgramId,
           serviceTemplate,
           deploy: false, // TODO: deprecated will remove
-          useMechMarketplace:
+          canUseMechMarketplace:
             STAKING_PROGRAMS[selectedAgentConfig.evmHomeChainId][
               selectedStakingProgramId
             ].mechType === MechType.Marketplace,
```

Comment:
@mohandast52 I don't know why and how this works (seems it's not even used by MW?) but if it is, we need to set mechType for `StakingProgramId.PearlBeta6` as MechType.Marketplace
can you either discuss with Jose if it's not used, or update to the correct value please?

Reply:
thanks a lot @mohandast52